configure.py

```diff
@@ -1418,6 +1421,15 @@ def find_ninja():
     'test/unit/cross_shard_barrier_test',
 ]) | pure_boost_tests
 
+
+COVERAGE_INST_FLAGS = ['-fprofile-instr-generate','-fcoverage-mapping',f'-fprofile-list=./{PROFILES_LIST_FILE_NAME}']
+if args.coverage:
+    for _ ,mode in filter(lambda m: m[0] != "coverage" ,modes.items()):
+        mode['cxx_ld_flags'] += ' ' + ' '.join(COVERAGE_INST_FLAGS)
+        mode['cxx_ld_flags'] = mode['cxx_ld_flags'].strip()
```

Comment:
to clear a leading space in case `cxx_ld_flags` was an empty string before adding the coverage flags.
I could leave it as is but wanted to have a clean `cxx_ld_flags` string.